discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -220,12 +221,18 @@ def attach_test_disk(self, node_name, disk_size, bootable=False, persistent=Fals
         if persistent:
             attach_flags |= libvirt.VIR_DOMAIN_AFFECT_CONFIG
 
+        wwn = ""
+
+        if with_wwn:
```

Comment:
maybe reduce these 4 lines to a single ternary:
`wwn = f"<wwn>0x{secrets.token_hex(8)}</wwn>" if with_wwn else ""`